src/Services/slidesServices.js

```diff
@@ -59,4 +68,4 @@ const DeleteSlide = () => {
   .catch(err => (err))
 }
 
-export {CreateSlide, EditSlide, GetAllSlides,GetAllSlidesById, DeleteSlide }
+export {CreateSlide, EditSlide, GetAllSlidesById, DeleteSlide, getSlides, getImagesSlides };
```

Comment:
please refactor all the code to use the same logic, or axios.then.catch or try{} catch{}

Reply:
also always return data, on delete, getbyId, return and create the methods are not returning anything

Reply:
Fixed